### PR TITLE
[Release] Download page for Mime4J 0.8.4

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Review by rouazana — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Comment by vttranlina** [comment]
I'm want to remind you that a lot of the links are invalid (404 not found).

**Comment by chibenwa** [comment]
I am aware of that, because I did not uploaded the corresponding content yet.

I will do tomorrow. And eventually proceed with the release.

File: src/site/xdoc/download.xml [hunk 1]
```diff
@@ -248,15 +248,15 @@ is found <a href='https://www.apache.org/licenses/exports/'>here</a>.
...
   &lt;/dependency&gt;
      </code>
     </pre>
 
     <p>Direct download link:
-      <a href="https://www.apache.org/dyn/closer.lua/james/mime4j/0.8.3/apache-mime4j-core-0.8.3.jar">(Jar)</a>
-      [<a href="https://downloads.apache.org/james/mime4j/0.8.3/apache-mime4j-core-0.8.3.jar.sha1">SHA-1</a>]
-      [<a href="https://downloads.apache.org/james/mime4j/0.8.3/apache-mime4j-core-0.8.3.jar.asc">PGP</a>]
+      <a href="https://www.apache.org/dyn/closer.lua/james/mime4j/0.8.4/apache-mime4j-core-0.8.4.jar">(Jar)</a>
+      [<a href="https://downloads.apache.org/james/mime4j/0.8.4/apache-mime4j-core-0.8.4.jar.sha1">SHA-1</a>]
```
Comment (chibenwa) [inline src/site/xdoc/download.xml:258]:
```suggestion
      [<a href="https://downloads.apache.org/james/mime4j/0.8.4/apache-mime4j-core-0.8.4.jar.sha512">SHA-512</a>]
```

File: src/site/xdoc/download.xml [hunk 2]
```diff
@@ -266,15 +266,15 @@ is found <a href='https://www.apache.org/licenses/exports/'>here</a>.
...
   &lt;/dependency&gt;
      </code>
     </pre>
 
     <p>Direct download link:
-      <a href="https://www.apache.org/dyn/closer.lua/james/mime4j/0.8.3/apache-mime4j-dom-0.8.3.jar">(Jar)</a>
-      [<a href="https://downloads.apache.org/james/mime4j/0.8.3/apache-mime4j-dom-0.8.3.jar.sha1">SHA-1</a>]
-      [<a href="https://downloads.apache.org/james/mime4j/0.8.3/apache-mime4j-dom-0.8.3.jar.asc">PGP</a>]
+      <a href="https://www.apache.org/dyn/closer.lua/james/mime4j/0.8.4/apache-mime4j-dom-0.8.4.jar">(Jar)</a>
+      [<a href="https://downloads.apache.org/james/mime4j/0.8.4/apache-mime4j-dom-0.8.4.jar.sha1">SHA-1</a>]
```
Comment (chibenwa) [inline src/site/xdoc/download.xml:276]:
```suggestion
      [<a href="https://downloads.apache.org/james/mime4j/0.8.4/apache-mime4j-dom-0.8.4.jar.sha512">SHA-512</a>]
```

File: src/site/xdoc/download.xml [hunk 3]
```diff
@@ -284,15 +284,15 @@ is found <a href='https://www.apache.org/licenses/exports/'>here</a>.
...
   &lt;/dependency&gt;
      </code>
     </pre>
 
     <p>Direct download link:
-      <a href="https://www.apache.org/dyn/closer.lua/james/mime4j/0.8.3/apache-mime4j-0.8.3-bin.zip">(Zip)</a>
-      [<a href="https://downloads.apache.org/james/mime4j/0.8.3/apache-mime4j-0.8.3-bin.zip.sha1">SHA-1</a>]
-      [<a href="https://downloads.apache.org/james/mime4j/0.8.3/apache-mime4j-0.8.3-bin.zip.asc">PGP</a>]
+      <a href="https://www.apache.org/dyn/closer.lua/james/mime4j/0.8.4/apache-mime4j-0.8.4-bin.zip">(Zip)</a>
+      [<a href="https://downloads.apache.org/james/mime4j/0.8.4/apache-mime4j-0.8.4-bin.zip.sha1">SHA-1</a>]
```
Comment (chibenwa) [inline src/site/xdoc/download.xml:294]:
```suggestion
      [<a href="https://downloads.apache.org/james/mime4j/0.8.4/apache-mime4j-0.8.4-bin.zip.sha512">SHA-512</a>]
```

File: src/site/xdoc/download.xml [hunk 4]
```diff
@@ -233,12 +233,12 @@ is found <a href='https://www.apache.org/licenses/exports/'>here</a>.
...
   
-    <p>Apache Mime4J 0.8.3 is the latest stable version.</p>
+    <p>Apache Mime4J 0.8.4 is the latest stable version.</p>
 
     <p>Sources:
-      <a href="https://www.apache.org/dyn/closer.lua/james/mime4j/0.8.3/james-mime4j-sources-0.8.3.zip">(Zip)</a>
-      [<a href="https://downloads.apache.org/james/mime4j/0.8.3/james-mime4j-sources-0.8.3.zip.sha1">SHA-1</a>]
-      [<a href="https://downloads.apache.org/james/mime4j/0.8.3/james-mime4j-sources-0.8.3.zip.asc">PGP</a>]
+      <a href="https://www.apache.org/dyn/closer.lua/james/mime4j/0.8.4/james-mime4j-sources-0.8.4.zip">(Zip)</a>
+      [<a href="https://downloads.apache.org/james/mime4j/0.8.4/james-mime4j-sources-0.8.4.zip.sha1">SHA-1</a>]
```
Comment (chibenwa) [inline src/site/xdoc/download.xml:240]:
```suggestion
      [<a href="https://downloads.apache.org/james/mime4j/0.8.4/james-mime4j-sources-0.8.4.zip.sha512">SHA-512</a>]
```

**Pull request merged**